### ISSUE TITLE
WeakMap requires new 

### ISSUE THREAD

**Issue by mblarsen** [fake-news-detector/extension#13]
Popup didn't seem to open after installing the Chrome extension.

<img width="162" alt="screen shot 2017-10-18 at 6 16 04 pm" src="https://user-images.githubusercontent.com/247048/31714515-378bbe62-b433-11e7-9d07-b9a83b7962a1.png">

Inspected the console and I see:

![screen shot 2017-10-18 at 6 16 41 pm](https://user-images.githubusercontent.com/247048/31714312-fba6660e-b432-11e7-84c7-2b53f1cc9609.png)

**Comment by rogeriochaves** [comment]
I'll check it out, thanks for the report!

**Comment by rogeriochaves** [comment]
Found it, this was caused by babel + uglifyjs during the compilation of `webextension-polyfill` I started using for chrome. I've tested it on chrome before publishing but not the minifed version, sorry!

Since this only runs on newer chrome and firefox, I've updated to use the uglifyjs version that understands es6 and removed babel entirely from the building process. Now babel is only used for mocha to understand the es6 `import`s.

Long story short, this was fixed by commit 48e6916 and a new 0.10 version is already on the way to the stores

**Comment by rogeriochaves** [comment]
Done, new version 0.10 released to chrome web store: https://chrome.google.com/webstore/detail/fake-news-detector/alomdfnfpbaagehmdokilpbjcjhacabk

<img width="182" alt="image" src="https://user-images.githubusercontent.com/792201/31737415-2f12aec4-b426-11e7-99da-5a43cae5108b.png">
